std/conv.d

```diff
@@ -2862,7 +2862,17 @@ unittest
 
     // min and max
     real r = to!real(to!string(real.min_normal));
-    assert(to!string(r) == to!string(real.min_normal));
+    version(NetBSD)
+    {
+        // NetBSD notice
+        // to!string returns 3.3621e-4932L. It is less than real.min_normal and it is subnormal value
+        // Simple C code 
+        //     long double rd = 3.3621e-4932L;
+        //     printf("%Le\n", rd);  
+        // has unexpected result: 1.681050e-4932 
```

Comment:
I confess that I'm not super familiar with the details of floating point stuff like min_normal, but this comment makes it seem like you're versioning out the test just because it's not passing on NetBSD rather than fixing the code so that it works on NetBSD, which definitely isn't a good thing.